SentinelOne- V2- 3.2.34 - Adding context data for command `sentinelone-get-power-query-results` (#36576)
demisto/content: Packs/SentinelOne/ReleaseNotes/3_2_34.md

#### Integrations

##### SentinelOne v2

> - Updated the ***sentinelone-get-power-query-results*** command to have powerquery output saved in context data.

Reviewer: ```suggestion
Added the *PowerQuery* context output to the ***sentinelone-get-power-query-results*** command.
```
